src/Element/ExecutionEngine/AutomationAction/Messenger/Messages/ElementBinMessage.php

```diff
@@ -14,13 +14,13 @@
  *  @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-namespace Pimcore\Bundle\StudioBackendBundle\Asset\ExecutionEngine\AutomationAction\Messenger\Messages;
+namespace Pimcore\Bundle\StudioBackendBundle\Element\ExecutionEngine\AutomationAction\Messenger\Messages;
 
 use Pimcore\Bundle\GenericExecutionEngineBundle\Messenger\Messages\AbstractExecutionEngineMessage;
 
 /**
  * @internal
  */
-final class AssetDeleteMessage extends AbstractExecutionEngineMessage
+final class ElementBinMessage extends AbstractExecutionEngineMessage
```

Comment:
Should we maybe think about about renaming it to RecycleBin or TrashBin? ElementBin is not that representativ? 
WDYT?